include/dsn/dist/failure_detector/failure_detector.h

```diff
@@ -105,7 +105,7 @@ class failure_detector :
 
     void register_master(::dsn::rpc_address target);
 
-    bool switch_master(::dsn::rpc_address from, ::dsn::rpc_address to);
+    bool switch_master(::dsn::rpc_address from, ::dsn::rpc_address to, uint32_t delay_milliseconds);
```

Comment:
the case is: if meta1 coredump, there is a time gap util another meta (if meta2) come on. During the time:
- replica-server/client will try to connect meta1, and get "Connection Refused" immediately
- then they will switch to the next one (if meta2), but now meta2 have not grabbed the lock and still thinks that meta1 is the master, and response to replica-server/client
- replica-server/client then switch master to meta1 again, and try to connect meta1 immediately, and again, they get "Connection Refused" immediately
- again and again, util meta2 becomes the master, and you can find from log that almost every millisecond there is a rpc try
After the patch, there will be one try every second when master switches.